apps/flame-defi/app/constants.ts

```diff
@@ -15,6 +15,11 @@ export enum TXN_STATUS {
   FAILED = "failed",
 }
 
```

Comment:
I know the above types were already here (TRADE_TYPE, TXN_STATUS) but they should really probably belong in `packages/flame-types/src/types.ts`

TOKEN_INPUTS should probably be moved to `swap/page.tsx` as it's only ever used there or in its components, so we are still enforcing dependency inversion correctly. 

In general, I don't think we need this constants.ts at all.